system-tests/projects/e2e/cypress/e2e/xhr.cy.js

```diff
@@ -157,18 +154,16 @@ describe('xhrs', () => {
     it('aborts', () => {
       cy.window()
       .then((win) => {
-        cy.route({
-          method: 'POST',
-          url: /users/,
-          response: { name: 'b' },
+        cy.intercept('POST', /users/, {
+          body: { name: 'b' },
           delay: 2000,
         }).as('createUser')
 
         cy.get('#create').click().then(() => {
           return win.location.href = '/index.html'
         })
 
-        cy.wait('@createUser').its('canceled').should('be.true')
+        cy.wait('@createUser').its('state').should('eq', 'Errored')
```

Comment:
How come the aborted state is now an errored state instead of canceled?

Reply:
intercept doesn't have a concept of canceled, so these are now marked as errored with the error message
>'Socket closed before finished writing response.'

Which maybe could be clearer but i'm not 100 percent sure that canceling the request is the only way we can get into this state.